docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -1431,6 +1431,11 @@ results_cache:
 # CLI flag: -query-frontend.max-retries-per-request
 [max_retries: <int> | default = 5]
 
+# (experimental) Time to wait between retries for a request that fails because
```

Comment:
(nit) Is it fair to mark something as experimental when it is enabled regardless and this is just a setting to tune it?

In my mind experimental is something new (and immature) you can turn on and off.

Reply:
I've changed the default value to preserve the existing behaviour (always retry immediately on failure).

I've marked it as experimental so that we can remove the flag altogether if this turns out to be a bad idea or doesn't work as expected - otherwise we'll need to go through a deprecation cycle to remove it.